From 8b5770142a3bad12c684f64bd79c8850ad93bf3c Mon Sep 17 00:00:00 2001
From: David Mandelberg <david@mandelberg.org>
Date: Fri, 4 Jan 2019 21:26:47 -0500
Subject: [PATCH 2/3] Mark GIT_DIR for export.

Before:
  yadm push <TAB>  # Completes filenames.

After:
  yadm push <TAB>  # Completes names of git remotes.
---
 completion/yadm.bash_completion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/completion/yadm.bash_completion b/completion/yadm.bash_completion
index 34f199f..60a71ff 100644
--- a/completion/yadm.bash_completion
+++ b/completion/yadm.bash_completion
@@ -18,7 +18,7 @@ if declare -F _git > /dev/null; then
       antepenultimate=${COMP_WORDS[COMP_CWORD-2]}
     fi
 
-    local GIT_DIR
+    local -x GIT_DIR
     # shellcheck disable=SC2034
     GIT_DIR="$(yadm introspect repo 2>/dev/null)"
 
-- 
2.20.1

